pennylane_lightning/core/src/measurements/MeasurementsBase.hpp

```diff
@@ -288,6 +275,7 @@ template <class StateVectorT, class Derived> class MeasurementsBase {
             sv.updateData(data_storage.data(), data_storage.size());
             obs.applyInPlaceShots(sv, eigenvalues, obs_wires);
             Derived measure(sv);
+            measure.set_DeviceSeed(this->DeviceSeed);
```

Comment:
I think lightning C++ is all CamelCase, so words are chained via capitalized letters and overall first letter is capitalized for nouns and not capitalized for verbs, i.e.
`MyCustomObject`, `StateVector`; `doSomething`, `applyInPlaceShots`

Reply:
```suggestion
            measure.setDeviceSeed(this->DeviceSeed);
```

Reply:
Done